packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -416,9 +415,6 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         const attributes = createAttributes(pkg, isRootDataStore, this.disableIsolatedChannels);
         addBlobToSummary(summarizeResult, dataStoreAttributesBlobName, JSON.stringify(attributes));
 
-        // Add GC details to the summary.
```

Comment:
Removing this will result in documents running older versions of runtime to re-run GC and summary for all data stores. It's expensive but it makes the code cleaner. Maybe we should keep it writing for couple of versions. Thoughts? @vladsud